### DISCUSSION TITLE
Activate circulation pump

### DISCUSSION THREAD

**Discussion by cyberthom42** [john30/ebusd#703] [Q&A]
My setup: arotherm plus 55/6, Hydraulikstation MEH 97/6, sensoComfort 720f, Internetmodul VR 920.
I have ebusd running on a raspberry using the ebus adapter, really great work. 
My question: Is there any way to enable and disable the circulation pump via ebus? 
I can read the status of the pump via "CircoPump".

**Comment by janphkre** [comment]
The control panel "ctlv2" of the SensoComfort always writes the message "SetActorState" on the SensoComfort base station "vr_71". This method contains the state of the actuators R* and depending on your wiring, one of them is the circulation pump. However, since the ctlv2 is always calling this method, your own calls may be overwritten by the control panel, so it makes more sense to change the timeslots of the circulation pump in the control panel. Please note, that the (hot water) circulation pump seems to be only set active by the ctlv2, if hot water is switched on. So if you want to run the circulation pump without hot water, you need to remove the hot water timeslots instead of deactivating hot water.

**Comment by cyberthom42** [comment]
I also figured out that the circulation pump only works if hot water is enabled via an appropriate time slot. This is even worse. My initial idea was to set the hot water time slot to noon to make sure enough PV is used for hot water. AT least I could fix this problem by enable hot water from 7-20h, but having a script running to adapt the HwcTempDesired and CylinderChargeHyst in a way that hot water will be loaded.
However I am still looking for a solution to enable circulation for e.g. 5-10 minutes every hour. I only have 3 time slots available. 
By the way, in my case I do not have a vr_71, this is only used if you have mixer circuits. I have a "Trinkwasserstation", the circulation pump will be enabled by that, controlled via the VRC720f.
